### ISSUE TITLE
Trouble making api request

### ISSUE THREAD

**Issue by robmontesinos** [rinov/YoutubeKit#2]
this library looks very promising with a lot of great work.

I made the following request in viewDidLoad to test:

```
        let request = ChannelListRequest(part: [.snippet], filter: .userName("RobMontesinos"))
        ApiSession.shared.send(request) { (result) in
            print("Response: \(result)")
            
            switch result {
            case .success(let response):
                print(response)
            case .failed(let error):
                print(error)
            }
        }
```

The following was returned:

Response: failed(Swift.DecodingError.keyNotFound(YoutubeKit.Snippet.ChannelList.CodingKeys.customURL, Swift.DecodingError.Context(codingPath: [YoutubeKit.ChannelList.(CodingKeys in _09738653794776D581197F11645E75D9).items, Foundation.(_JSONKey in _12768CA107A31EF2DCE034FD75B541C9)(stringValue: "Index 0", intValue: Optional(0)), YoutubeKit.Channel.(CodingKeys in _09738653794776D581197F11645E75D9).snippet], debugDescription: "No value associated with key customURL (\"customUrl\").", underlyingError: nil)))
keyNotFound(YoutubeKit.Snippet.ChannelList.CodingKeys.customURL, Swift.DecodingError.Context(codingPath: [YoutubeKit.ChannelList.(CodingKeys in _09738653794776D581197F11645E75D9).items, Foundation.(_JSONKey in _12768CA107A31EF2DCE034FD75B541C9)(stringValue: "Index 0", intValue: Optional(0)), YoutubeKit.Channel.(CodingKeys in _09738653794776D581197F11645E75D9).snippet], debugDescription: "No value associated with key customURL (\"customUrl\").", underlyingError: nil))

It seems that the Codable protocol is failing somehow. Any idea of what I'm doing wrong? Thank you.

**Comment by rinov** [comment]
Thank you for your report! @robmontesinos 
I noticed in your report, `CustomURL` should be optional type, so i've just fixed it at version 0.1.2.
Please try again with following version. 

`$pod repo update` 
and, add this in your Podfile.
`'YoutubeKit', '~> 0.1.2'`

Than you.

FYI: This is the response in my environment.

```

ChannelList(etag: "\"S8kisgyDEblalhHF9ooXPiFFrkc/w5zOuWz09yInPwNZxC26OaR2--M\"", items: [YoutubeKit.Channel(brandingSettings: nil, contentDetails: nil, etag: "\"S8kisgyDEblalhHF9ooXPiFFrkc/tO8G-Kr8SBxYTw_2E--9HyAZF0s\"", id: "UCJk88t9iXjY3TmG9CV8Qxcw", kind: "youtube#channel", snippet: Optional(YoutubeKit.Snippet.ChannelList(customURL: nil, description: "Roberto Montesinos. Professional Actor. iOS Developer", localized: YoutubeKit.Localized(description: "Roberto Montesinos. Professional Actor. iOS Developer", title: "Roberto Montesinos"), publishedAt: "2006-11-13T18:40:21.000Z", thumbnails: YoutubeKit.Thumbnails.Default(url: nil, height: nil, width: nil), title: "Roberto Montesinos")), statistics: nil, topicDetails: nil)], kind: "youtube#channelListResponse", pageInfo: YoutubeKit.PageInfo(resultsPerPage: 5, totalResults: 1), nextPageToken: nil, prevPageToken: nil)
```

**Comment by rinov** [comment]
Fixed by v0.1.2
